inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -145,46 +138,43 @@ def simulate_async(
                                       resource_pool_id=resource_pool_id,
                                       commands=commands)
 
-    @singledispatchmethod
-    def gen_config(self, simulator: Simulator):
-        raise ValueError(
-            f"Simulator not supported for `{self.__class__.__name__}` scenario."
-        )
+    def get_commands(self):
+        """Returns the commands for the simulation."""
 
-    @singledispatchmethod
-    def gen_aux_files(self, simulator: Simulator, input_dir: str):
-        raise ValueError(
-            f"Simulator not supported for `{self.__class__.__name__}` scenario."
-        )
+        commands_template_path = os.path.join(SCENARIO_TEMPLATE_DIR,
+                                              GROMACS_TEMPLATE_INPUT_DIR,
+                                              COMMANDS_TEMPLATE_FILE_NAME)
 
-    @singledispatchmethod
-    def get_config_filename(self, simulator: Simulator):  # pylint: disable=unused-argument
-        raise ValueError(
-            f"Simulator not supported for `{self.__class__.__name__}` scenario."
-        )
+        with tempfile.NamedTemporaryFile() as commands_file:
```

Comment:
This is awesome and will solve the problem of having always the commands.json commited by mistake. Thanks Fábio :)